kubernetes/cray-keycloak/values.yaml

```diff
@@ -156,140 +156,185 @@ keycloak:
-      traffic.sidecar.istio.io/excludeInboundPorts: '7600'
-      traffic.sidecar.istio.io/excludeOutboundPorts: '7600'
-    image:
-      repository: artifactory.algol60.net/csm-docker/stable/docker.io/jboss/keycloak
-      tag: 9.0.0
-    # NOTE: When upgrading this should be able to be removed. -XX:+UseContainerSupport is the default in chart version 7.3.0
-    extraEnv: |
-      - name: JAVA_OPTS
-        value: -XX:+UseContainerSupport -XX:MaxRAMPercentage=50.0 -Djava.net.preferIPv4Stack=true -Djboss.modules.system.pkgs=org.jboss.byteman -Djava.awt.headless=true
-      - name: PROXY_ADDRESS_FORWARDING
-        value: "true"
-    basepath: keycloak
-    username: admin
-    existingSecret: keycloak-master-admin-auth
-    serviceAccount:
-      name: jobs-watcher
-    # When nodeAffinity isn't needed anymore, note that the podAntiAffinity is
-    # copied from the defaults so `affinity` can be removed.
-    priorityClassName: csm-high-priority-service
-    affinity: |
-      podAntiAffinity:
-        requiredDuringSchedulingIgnoredDuringExecution:
-          - labelSelector:
-              matchLabels:
-                {{- include "keycloak.selectorLabels" . | nindent 10 }}
-              matchExpressions:
-                - key: role
-                  operator: NotIn
-                  values:
-                    - test
-            topologyKey: kubernetes.io/hostname
-      nodeAffinity:
-        requiredDuringSchedulingIgnoredDuringExecution:
-          nodeSelectorTerms:
-          - matchExpressions:
-            - key: no_external_access
-              operator: NotIn
-              values:
-              - "True"
-    # cray-postgres-operator provides the secret, etc.
-    persistence:
-      dbVendor: postgres
-      dbName: service_db
-      dbHost: keycloak-postgres
-      existingSecret: service-account.keycloak-postgres.credentials
-      existingSecretPasswordKey: password
-      existingSecretUsernameKey: username
-    # This is copied from cray-services
-    extraInitContainers: |
-      - name: keycloak-wait-for-postgres
-        image: {{ .Values.kubectl.image.repository }}:{{ .Values.kubectl.image.tag }}
-        imagePullPolicy: {{ .Values.kubectl.image.pullPolicy }}
-        command:
-        - /bin/sh
-        - -c
-        - |
-          while true; do
-            JOB_CONDITION="$(kubectl get jobs -n services -l app.kubernetes.io/name=keycloak-wait-for-postgres -o jsonpath='{.items[0].status.conditions[0].type}')"
-            JOB_CONDITION_RC=$?
-            if [ $JOB_CONDITION_RC -eq 0 ]; then
-              if [ "$JOB_CONDITION" == 'Complete' ]; then
-                echo "Completed"
-                break
-              fi
-              echo "Waiting for the keycloak-wait-for-postgres job in the services namespace to complete, current condition is $(kubectl get jobs -n services -l app.kubernetes.io/name=keycloak-wait-for-postgres -o jsonpath='{.items[0].status}')"
```

Comment:
Are these lines 228 - 235 that were removed from under keycloak still need to be included in the new structure?